tests/components/ipp/test_config_flow.py

```diff
@@ -364,17 +376,20 @@ async def test_full_zeroconf_tls_flow_implementation(
 
     assert result["step_id"] == "zeroconf_confirm"
     assert result["type"] == RESULT_TYPE_FORM
-    assert result["description_placeholders"] == {CONF_NAME: "EPSON123456"}
+    assert result["description_placeholders"] == {CONF_NAME: "EPSON XP-6000 Series"}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5373293)